### PR TITLE
Updated profile fields to display current data

### PR DESCRIPTION
**Pull request by TillaTheHun0**
This PR consist of a service that transforms a firebase ref or query into a firebaseObject. Also, name and description fields reflect what is currently stored in firebase and can be edited. However, by design, values are not persisted until the save button is clicked, which uses the built-in save functionality of the firebaseObject. This offers more robustness and security.

### CODE REVIEW

File: www/js/services.js [hunk 1]
```diff
@@ -1,13 +1,33 @@
...
+       return $firebaseObject(firebaseRef); 
+    }
+    return transform; 
+})
+
+/**
+ * Spinner that blocks user interaction and displays spinning icon.
+ * Can be used when loading data.
+ */
 .factory('Spinner', ['$ionicLoading', function($ionicLoading){
```
Comment (bmykietka) [inline www/js/services.js:31]: Should rename this to "SpinnazBlingBling"

**Comment by TillaTheHun0** [comment]
you're right. What was I thinking when I decided to write non-esoteric code. :sob:

File: www/js/controllers.js [hunk 2]
```diff
@@ -119,21 +119,24 @@ angular.module('app.controllers', [])
     }
 })
    
-.controller('profileCtrl',function($scope, firebaseAuth, $state, rootRef, $rootScope, $ionicLoading, Spinner) {
-	   $scope.signOut = function(){
+.controller('profileCtrl', function($scope, firebaseAuth, $state, rootRef, $rootScope, $ionicLoading, Spinner, firebaseObject, $timeout) {
+	   var authObject = firebaseAuth.$getAuth();
```
Comment (bmykietka) [inline www/js/controllers.js:123]: Indentationness.

Comment (TillaTheHun0) [inline www/js/controllers.js:123]: :+1:

**Pull request merged**